src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,23 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  font-weight: 400;
```

Comment:
Default value, can be removed

Reply:
I tried. font-size: 10px; is not default.
So I removed font-weight: 400;

Reply:
I meant only font-weight 🙂